File: app/src/main/java/com/example/android_project/Models/Categorias.java
package com.example.android_project.Models;

public enum Categorias {
    ACCION("Acción"),
    COMEDIA("Comedia"),
    DRAMA("Drama"),
    CIENCIA_FICCION("Ciencia Ficción"),
    ANIMACION("Animación"),
    TERROR("Terror");

    private final String nombre;

    Categorias(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }
}

File: app/src/main/java/com/example/android_project/Models/Categorias.kt
package com.example.android_project.Models

enum class Categorias(val nombre: String) {
    ACCION("Acción"),
    COMEDIA("Comedia"),
    DRAMA("Drama"),
    CIENCIA_FICCION("Ciencia Ficción"),
    ANIMACION("Animación"),
    TERROR("Terror");

    override fun toString(): String {
        return "$nombre\n"
    }
}